src/network/networkd-ndisc.c

```diff
@@ -385,17 +384,6 @@ static int ndisc_router_process_autonomous_prefix(Link *link, sd_ndisc_router *r
                 address->lifetime_valid_usec = lifetime_valid_usec;
                 address->lifetime_preferred_usec = lifetime_preferred_usec;
 
-                /* See RFC4862, section 5.5.3.e. But the following logic is deviated from RFC4862 by
-                 * honoring all valid lifetimes to improve the reaction of SLAAC to renumbering events.
-                 * See draft-ietf-6man-slaac-renum-02, section 4.2. */
-                r = address_get(link, address, &e);
-                if (r > 0) {
```

Comment:
Note: this check never passes anyway as of commit 3b6a3bdebfb5 ("network: use NetworkConfigSource/State to manage addresses and routes"), since that commit changed `address_get()` to only ever return `-ENOENT` or 0. So getting rid of this probably also removes a bug.

Reply:
Ouch, indeed.